### PR TITLE
Adding tails server and tunnel agents to docker compose file

### PR DESCRIPTION
**Pull request by anwalker293**
Adding tails server and tunnel agents to docker compose file :)

Tested and ran fine under

`docker-compose build`
`docker-compose run tests`
`docker-compose down`

Signed-off-by: Alexandra N. Walker ☀️️

### CODE REVIEW

File: docker-compose.yml [hunk 1]
```diff
@@ -117,3 +161,25 @@ services:
...
+      - ./controller:/usr/src/app/controller:z
+      - ./tests:/usr/src/app/tests:z
+    entrypoint: "poetry run pytest"
+    depends_on:
+      echo:
+        condition: service_healthy
+      alice:
+        condition: service_healthy
+      bob:
+        condition: service_healthy
```
Comment (dbluhm) [inline docker-compose.yml:185]: There's two tests services?

File: docker-compose.yml [hunk 2]
```diff
@@ -95,6 +125,20 @@ services:
...
+    ports:
+      - 6543:6543
+    environment:
+      - GENESIS_URL=https://raw.githubusercontent.com/Indicio-tech/indicio-network/master/genesis_files/pool_transactions_testnet_genesis
+    command: >
+      tails-server
+      --host 0.0.0.0
+      --port 6543
+      --storage-path /tmp/tails-files
+      --log-level INFO
```
Comment (dbluhm) [inline docker-compose.yml:141]: The tails server needs to run on the bcgov image; the tunnel for the tails server will need to be a separate service with that image used.

**Review by dbluhm — CHANGES_REQUESTED** [review]
Couple fixes needed :slightly_smiling_face:

File: docker-compose.yml (hunk 1 above)
Comment (anwalker293) [reply docker-compose.yml:185]: whoops! Thanks! :)

File: docker-compose.yml (hunk 2 above)
Comment (anwalker293) [reply docker-compose.yml:141]: *thumbs up*

**Comment by anwalker293** [comment]
Yay! I think I did this successfully @dbluhm

**Review by dbluhm — APPROVED** [review]
Looks good! Thanks!

**Pull request merged**